coalib/results/TextPosition.py

```diff
@@ -21,6 +23,10 @@ def __init__(self, line: (int, None)=None, column: (int, None)=None):
         if line is None and column is not None:
             raise ValueError('A column can only be set if a line is set.')
 
+        if line == 0 or column == 0:
+            logging.warning('line or column offset should not be 0. '
+                            'Please report this to developers.')
```

Comment:
To be useful, we need to report to the user which bear it was.  The only way to do that is raise an exception here, and catch it somewhere higher.

Would be good to have separate exception for line 0 and column 0, to make it easier to find the source of the problem.